LLM_Compressor_0.9.0.md

```diff
@@ -0,0 +1,125 @@
+# LLM Compressor 0.9.0 release notes
+
+This LLM Compressor 0.9.0 release adds support for MXFP4 compression, provides expanded quantization support, model compatibility improvements, and many other performance, workflow, and code optimizations.
+This release also adds experimental support for R3 transforms and removes support for Python 3.9.
+
+<!-- https://issues.redhat.com/browse/INFERENG-1889 -->
+<!-- https://issues.redhat.com/browse/INFERENG-1890 -->
+## Added support for MXFP4 compression ✨
+
+LLM Compressor and compressed-tensors now support MXFP4 quantization and calibration of MXFP4 scales.
+To use MXFP4 quantization, quantize models with the new MXFP4 preset scheme.
+The `MXFP4PackedCompressor` class compresses and saves the model, packing both weights and scales as uint8 integers.
+MXFP4 quantization is experimental pending validation with vLLM. 
+
+Two new fields, `scale_dtype` and `zp_dtype`, have been added to the `QuantizationArgs` class: 
+
+- `scale_dtype`: When set to `None`, scales are saved using the default dense data type.
+When specified, scales are compressed using the provided data type.
+For example, NVFP4 saves scales as FP8, while MXFP4 saves them as uint8.
+This data type is reflected in the model config.
+- `zp_dtype`: Set to `None` for symmetric models.
+For asymmetric models, this specifies the data type used to save zero-point values.
+
+See the [quant_args.py#L157](https://github.com/vllm-project/compressed-tensors/blob/797d3019ef6867362796f412980547c74551f369/src/compressed_tensors/quantization/quant_args.py#L157) implementation for details.
+
+<!-- https://issues.redhat.com/browse/INFERENG-2163 -->
+## Added experimental R3 transforms support
+
+LLM Compressor now has expermental support for applying transforms to attention in the style of the R3 rotation used in SpinQuant models.
+R3 transforms can potentially increase accuracy recovery for extreme attention quantization.
+R3 rotations are not yet supported in vLLM.
```

Comment:
We will have a link to an example here: `experimental/attention/llama3_attention_r3_nvfp4.py` (still has to be merged)